SerialPrograms/Source/PokemonSV/Programs/PokemonSV_Navigation.h

```diff
@@ -67,6 +67,10 @@ void leave_phone_to_overworld(const ProgramInfo& info, ConsoleHandle& console, B
 //  by other map icons on the most zoomed-in level of the map.
 void fly_to_closest_pokecenter_on_map(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context);
 
+// Attempt to escape being stuck on a wall.
+// Repeatedly center camera and try to backwards jump off.

```

Comment:
Since the end state here is the map being open, we should include that in the function name somehow.